modules/jupyter-widget/src/utils.js

```diff
@@ -67,3 +79,10 @@ export function initDeck({mapboxApiKey, container, jsonInput, tooltip, onComplet
     });
   });
 }
+
+function wrapWarn(warnFunction, messageHandler) {
```

Comment:
Call it `wrapFunction` or `injectFunction`. There is no logic specific to `warn`.